packages/lib/checkRateLimitAndThrowError.ts

```diff
@@ -2,18 +2,29 @@ import { HttpError } from "./http-error";
 import type { RateLimitHelper } from "./rateLimit";
 import { rateLimiter } from "./rateLimit";
 
-export async function checkRateLimitAndThrowError({
-  rateLimitingType = "core",
-  identifier,
-  onRateLimiterResponse,
-  opts,
-}: RateLimitHelper) {
-  const response = await rateLimiter()({ rateLimitingType, identifier, opts });
-  if (onRateLimiterResponse) onRateLimiterResponse(response);
-  const { success, reset } = response;
-  if (!success) {
+export async function checkRateLimitAndThrowError(req: RateLimitHelper | RateLimitHelper[]) {
```

Comment:
could be cleaned up but for backwards compatibility I just changed it to accept both a single req or an array